.github/workflows/pythontests.yml

```diff
@@ -20,16 +20,10 @@ on:
 
 jobs:
   build:
-    runs-on: >-
-      ${{ contains(fromJson(
-          '["3.6"]'
-      ), matrix.python-version) && 'ubuntu-20.04' || 'ubuntu-latest' }}
+    runs-on: ubuntu-latest
```

Comment:
Should we pin this to `ubuntu-22.04` to avoid surprises when the next version comes out?

Reply:
No, otherwise we only have even worse surprises when ubuntu-22.04 is suddenly no longer supported and we have to hurry to replace it.

Reply:
Ack